src/custom-select/index.ts

```diff
@@ -17,41 +17,39 @@ class CustomSelect {
         this.$select.onSelect(event.item.i);
       }
       this.update({ isActive: false });
-      this.refs.toggle.blur();
     }
   };
   isActive: boolean = false;
 
+  init() {
+    utils.WINDOW.addEventListener('click', this.onClickDeactivate);
```

Comment:
We'll be doing this in separate pull requests. In the meantime, we've updated `@storefront/core` and we're using `utils.WINDOW.document()` for now.